src/javaRosa.js

```diff
@@ -941,7 +941,7 @@ define([
         widget.setItextValue = function (value) {
             var itextItem = widget.getItextItem();
             if (itextItem) {
-                if (!value) {
+                if (!value && !mug.options.canHaveBlankLabel) {
                     value = widget.getPlaceholder();
```

Comment:
This is reverting to old behavior for groups, which means that if there is a blank label, then we will not automatically update it with the placeholder. I think if you create a group the placeholder will still be there unless you manually blank it out. This was the previous behavior before the bug was logged